akka-actor-typed/src/main/scala/akka/actor/typed/internal/WithMdcBehaviorInterceptor.scala

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.internal
+
+import scala.collection.immutable.HashMap
+import akka.actor.typed.Behavior.DeferredBehavior
+import akka.actor.typed.internal.adapter.AbstractLogger
+import akka.actor.typed.javadsl.Behaviors
```

Comment:
any reason for using javadsl instead of scaladsl here?

Reply:
Nop. IntelliJ made me do it. I'll fix.